src/components/Dialogs/Dialogs.jsx

```diff
@@ -1,39 +1,57 @@
 import React from "react";
+import { NavLink } from "react-router-dom";
+
+import { ROUTES, I18N } from "../core/constants";
 
 import s from "./Dialogs.module.css";
 
-const Dialogs = (props) => {
+const DialogItem = props => {
+  let path = ROUTES.MESSAGES + "/" + props.id;
+
+  return (
+    <div className={s.dialog + " " + s.active}>
+      <NavLink to={path}>{props.name}</NavLink>
+    </div>
+  );
+};
+
+const Message = props => {
+  return <div className={s.dialogs}>{props.message}</div>;
+};
+
+const Dialogs = props => {
+  let dialogsData = [
```

Comment:
Consider defaulting to use of "const" instead of "let" with objects, arrays and functions.
They are essentially object-like structures under the hood and therefore they are "passed by link".
So when you're using "const" with them, you're ensuring that only the contents of the object in this variable could be changed.

Reply:
done